json/spell-icon-migration.json

```diff
@@ -411,20 +411,20 @@
   "icons/magic/light/beam-rays-orange-purple-small.webp": "icons/magic/light/beams-rays-orange-purple-small.webp",
   "icons/magic/light/beam-rays-orange-purple.webp": "icons/magic/light/beams-rays-orange-purple.webp",
   "icons/magic/light/beam-rays-orange-purple-large.webp": "icons/magic/light/beams-rays-orange-purple-large.webp",
-  "icons/magic/light/explosion-star-large-blue.webp": "icons/magic/light/explosion-star-blue-large.webp",
-  "icons/magic/light/explosion-star-small-blue.webp": "icons/magic/light/explosion-star-blue-small.webp",
-  "icons/magic/light/explosion-star-large-blue-yellow.webp": "icons/magic/light/explosion-star-blue-yellow-large.webp",
-  "icons/magic/light/explosion-star-small-blue-yellow.webp": "icons/magic/light/explosion-star-blue-yellow-small.webp",
-  "icons/magic/light/explosion-star-large-teal.webp": "icons/magic/light/explosion-star-teal-large.webp",
-  "icons/magic/light/explosion-star-small-teal.webp": "icons/magic/light/explosion-star-teal-small.webp",
-  "icons/magic/light/explosion-star-large-teal-purple.webp": "icons/magic/light/explosion-star-teal-purple-large.webp",
-  "icons/magic/light/explosion-star-small-teal-purple.webp": "icons/magic/light/explosion-star-teal-purple-small.webp",
-  "icons/magic/light/explosion-star-large-orange.webp": "icons/magic/light/explosion-star-orange-large.webp",
-  "icons/magic/light/explosion-star-small-orange.webp": "icons/magic/light/explosion-star-orange-small.webp",
-  "icons/magic/light/explosion-star-large-orange-purple.webp": "icons/magic/light/explosion-star-orange-purple-large.webp",
-  "icons/magic/light/explosion-star-small-orange-purple.webp": "icons/magic/light/explosion-star-orange-purple-small.webp",
-  "icons/magic/light/explosion-star-large-pink.webp": "icons/magic/light/explosion-star-pink-large.webp",
-  "icons/magic/light/explosion-star-small-pink.webp": "icons/magic/light/explosion-star-pink-small.webp",
+  "icons/magic/light/explosion-star-blue-large.webp": "icons/magic/light/explosion-star-large-blue.webp",
```

Comment:
I can see in both v10 and v11 that the icon is called `explosion-star-blue-large`, so this change would map a correct icon filename to an incorrect filename, no? It looks like the original line was correct. What am I missing here?

Reply:
I think I picked the first line which just happened to be a mistake. The two `explosion-star-blue` lines need to be reverted, but I checked the other lines and they all seem to be correct.